### PR TITLE
[Storage] Delete file and folder (ipfs and chainsafe types) inside a bucket test

### LINKED ISSUE #2227: [Storage] Add delete file and folder inside a bucket test
As described add the following coverage:

- Delete file in IPFS bucket
- Delete folder in IPFS bucket
- Delete file in Chainsafe bucket
- Delete folder in Chainsafe bucket

### PR DESCRIPTION
**Pull request by juans-chainsafe**
closes #2227

### CODE REVIEW

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts [hunk 1]
```diff
@@ -431,5 +432,91 @@ describe("Bucket management", () => {
...
+      // delete the file
+      bucketContentsPage.deleteFileOrFolder()
+
+      deletionCompleteToast.body().should("be.visible")
+      deletionCompleteToast.closeButton().click()
+      bucketContentsPage.fileItemRow().should("have.length", 0)
+    })
+
+    it("can delete a folder inside the ipfs bucket", () => {
+      const ipfsBucketName = `chainsafe bucket ${Date.now()}`
```
Comment (asnaith) [inline packages/storage-ui/cypress/tests/bucket-management.cy.ts]:
```suggestion
      const ipfsBucketName = `ipfs bucket ${Date.now()}`
```

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts [hunk 2]
```diff
@@ -431,5 +432,91 @@ describe("Bucket management", () => {
...
+      // go inside the bucket
+      bucketsPage.bucketItemRow().should("have.length", 1)
+      bucketsPage.bucketItemName().dblclick()
+
+      // create a folder inside the bucket
+      bucketContentsPage.createNewFolder(folderName)
+
+      // delete the folder
+      bucketContentsPage.deleteFileOrFolder()
+
```
Comment (asnaith) [inline packages/storage-ui/cypress/tests/bucket-management.cy.ts]:
```suggestion
```

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts [hunk 3]
```diff
@@ -431,5 +432,91 @@ describe("Bucket management", () => {
...
+      // go inside the bucket
+      bucketsPage.bucketItemRow().should("have.length", 1)
+      bucketsPage.bucketItemName().dblclick()
+
+      // upload a file
+      bucketContentsPage.uploadFileToBucket("../fixtures/uploadedFiles/logo.png")
+
+      // delete the file
+      bucketContentsPage.deleteFileOrFolder()
+
```
Comment (asnaith) [inline packages/storage-ui/cypress/tests/bucket-management.cy.ts]:
```suggestion
```

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts [hunk 4]
```diff
@@ -431,5 +432,91 @@ describe("Bucket management", () => {
...
+      // go inside the bucket
+      bucketsPage.bucketItemRow().should("have.length", 1)
+      bucketsPage.bucketItemName().dblclick()
+
+      // create a folder inside the bucket
+      bucketContentsPage.createNewFolder(folderName)
+
+      // delete the folder
+      bucketContentsPage.deleteFileOrFolder()
+
```
Comment (asnaith) [inline packages/storage-ui/cypress/tests/bucket-management.cy.ts]:
```suggestion
```

File: packages/storage-ui/cypress/tests/bucket-management.cy.ts [hunk 5]
```diff
@@ -431,5 +432,91 @@ describe("Bucket management", () => {
...
+      // go inside the bucket
+      bucketsPage.bucketItemRow().should("have.length", 1)
+      bucketsPage.bucketItemName().dblclick()
+
+      // upload a file
+      bucketContentsPage.uploadFileToBucket("../fixtures/uploadedFiles/logo.png")
+
+      // delete the file
+      bucketContentsPage.deleteFileOrFolder()
+
```
Comment (asnaith) [inline packages/storage-ui/cypress/tests/bucket-management.cy.ts]:
```suggestion
```

**Review by asnaith — APPROVED** [review]
@juans-chainsafe Looks great, thanks for this additional test coverage. 

I left a few suggestions for nits but otherwise, this looks good to go 😃

**Review by tanmoyAtb — APPROVED** [review]
Looks green to go !

**Pull request merged**